spec/index.bs

```diff
@@ -2396,6 +2397,29 @@ namespace GPUTextureUsage {
 };
 </script>
 
+<dl dfn-type=dict-member dfn-for=GPUTextureDescriptor>
+    : <dfn>viewFormats</dfn>
+    ::
+        Specifies what view {{GPUTextureViewDescriptor/format}} values will be allowed when calling
+        {{GPUTexture/createView()}} on this texture.
```

Comment:
should we also say that copying to/from these formats is allowed?

Reply:
Copying between srgb and non-srgb is always allowed (#2329), so we don't need anything more than that until viewFormats can express something beyond that (https://github.com/gpuweb/gpuweb/issues/842#issuecomment-641616837).